android/src/main/java/com/taskrabbit/zendesk/RNZendeskChatModule.java

```diff
@@ -282,10 +342,38 @@ public void startChat(ReadableMap options) {
 
     @ReactMethod
     public void registerPushToken(String token) {
-        PushNotificationsProvider pushProvider = Chat.INSTANCE.providers().pushNotificationProvider();
+        PushNotificationsProvider pushProvider = Chat.INSTANCE.providers().pushNotificationsProvider();
```

Comment:
is this update right? I got the code from the [docs](https://developer.zendesk.com/embeddables/docs/chat-sdk-v-2-for-android/push_notifications) and is says that the method is `pushNotificationProvider()` without the `s`.

```java
PushNotificationsProvider pushProvider = Chat.INSTANCE.providers().pushNotificationProvider();

if (pushProvider != null) {
    pushProvider.registerPushToken("YOUR FIREBASE TOKEN");
}
```

Reply:
Ok, I just checked it and it's right.

Reply:
it's works thank you @gabrielperales

Reply:
I'm glad you make it work @petchkubb 👍 . It was working for you in android? I'm not able to receive push notifications from the chat in my android App. Did you configure something I could be missing or something?